contracts/test/_fixture.js

```diff
@@ -1282,9 +1282,13 @@ async function hackedVaultFixture() {
   });
 
   const evilDAI = await ethers.getContract("MockEvilDAI");
+  const latestBlock = await ethers.provider.getBlock();
+  const maxStaleness = latestBlock.timestamp + 24 * 60 * 60;
```

Comment:
Same comment on using the timestamp for something that's a duration. Better to just use a really big number.